dotnet-conf-2022/index.html

```diff
@@ -0,0 +1,592 @@
+										available at <a style="color: #323eee;"
+											href="https://www.contributor-covenant.org/version/2/0/code_of_conduct.html"
+											target="_blank">https://www.contributor-covenant.org/version/2/0/code_of_conduct.html</a>.
+										While attending the event, you must follow this Code of Conduct. If you have any
+										questions or feedback, please contact one of the organizers.
+									</p>
+								</div>
+							</div>
+							<!-- End Feature Content -->
+						</div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<!-- End Start Code of Conduct -->
+		<!-- Start Sponsors -->
+		<section id="mu-sponsors">
+			<div class="container">
+				<div class="row">
+					<div class="col-md-12">
+						<div class="mu-sponsors-area">
+							<div class="mu-title-area">
+								<h2 class="mu-title">Sponsored by</h2>
+								<p>Thanks a ton to our sponsors for the awesome goodies and support!!!</p>
+							</div>
+
+							<!-- Start spnonsors brand logo -->
+
+							<div class="mu-sponsors-content">
+								<div class="row">
+									<div class="col-md-3">
+										<a href="https://www.microsoft.com" target="_blank">
+											<img src="assets/images/Microsoft.png" />
+										</a>
+									</div>
+									<div class="col-md-3">
+										<a href="https://dotnetfoundation.org/" target="_blank">
+											<img src="assets/images/dotnetfoundationhorizontal.svg" />
+										</a>
+									</div>
+									<div class="col-md-3">
+										<a href="https://www.thetechplatform.com" target="_blank">
+											<img src="assets/images/TheTechPlatform.PNG" />
+										</a>
+									</div>
+									<div class="col-md-3">
+										<a href="https://jetbrains.com" target="_blank">
+											<img src="assets/images/jetbrains.svg" />
+										</a>
+									</div>
+
+								</div>
+
+							</div>
+							<br />
+							<div class="mu-title-area" style="margin-top:6%;">
+								<h2 class="mu-title">Our .NET Community User Groups</h2>
+							</div>
+							<div class="row">
+								<div class="col-md-6" style="text-align:center;margin-top:4%;">
+									<div class="row">
+										<img src="assets/images/BDOT.png" style="width:150px;" />
+									</div>
+									<div class="row">
+										<div class="mu-social-media">
+											<a href="https://www.facebook.com/groups/BDotNet/" target="_blank"><i
+													class="fa fa-facebook"></i></a>
+											<a href="https://www.meetup.com/BDotNetUG/" target="_blank"><i
+													class="fa fa-meetup"></i></a>
+											<a href="https://www.youtube.com/user/BdotnetTV" target="_blank"><i
+													class="fa fa-youtube"></i></a>
+											<a href="https://twitter.com/bdotnetug" target="_blank"><i
+													class="fa fa-twitter"></i></a>
+										</div>
+									</div>
+								</div>
+								<div class="col-md-6" style="text-align:center;margin-top:4%;">
+									<div class="row">
+										<h1>Dot Net Learners House</h1>
```

Comment:
Dotnet learner's house related details can be removed as well

Reply:
Dotnet learner's house related details removed